src/components/views/WaiterOrderNew/WaiterOrderNew.js

```diff
@@ -1,10 +1,383 @@
+          },
+        },
+      },
 
-export default WaiterOrderNew;
+      toppings: {
+        label: 'Toppings',
+        type: 'checkboxes',
+        options: {
+          olives: {
+            label: 'Olives',
+            price: 2,
+            default: true,
+          },
+          redPeppers: {
+            label: 'Red peppers',
+            price: 2,
+            default: true,
+          },
+          greenPeppers: {
+            label: 'Green peppers',
+            price: 2,
+            default: true,
+          },
+          mushrooms: {
+            label: 'Mushrooms',
+            price: 2,
+            default: true,
+          },
+          basil: {
+            label: 'Fresh basil',
+            price: 2,
+            default: true,
+          },
+          salami: {
+            label: 'Salami',
+            price: 3,
+          },
+        },
+      },
+      crust: {
+        label: 'pizza crust',
+        type: 'select',
+        options: {
+          standard: {
+            label: 'standard',
+            price: 0,
+            default: true,
+          },
+          thin: {
+            label: 'thin',
+            price: 2,
+          },
+          thick: {
+            label: 'thick',
+            price: 2,
+          },
+          cheese: {
+            label: 'cheese in edges',
+            price: 5,
+          },
+          wholewheat: {
+            label: 'wholewheat',
+            price: 3,
+          },
+          gluten: {
+            label: 'with extra gluten',
+            price: 0,
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'salad',
+    name: `Nonno Alberto's Salad`,
+    price: 9,
+    params: {
+      ingredients: {
+        label: 'Ingredients',
+        type: 'checkboxes',
+        options: {
+          cucumber: {
+            label: 'Cucumber',
+            price: 1,
+            default: true,
+          },
+          tomatoes: {
+            label: 'Tomatoes',
+            price: 1,
+            default: true,
+          },
+          black_olives: {
+            label: 'Olives',
+            price: 1,
+            default: true,
+          },
+          feta: {
+            label: 'Feta cheese',
+            price: 1,
+          },
+          cheese: {
+            label: 'Cheese',
+            price: 1,
+          },
+          herbs: {
+            label: 'Fresh herbs',
+            price: 1,
+            default: true,
+          },
+          pepper: {
+            label: 'Black pepper',
+            price: 1,
+          },
+        },
+      },
+    },
+  },
+];
+
+export default function WaiterOrderNew() {
+  const [value, setValue] = React.useState('');
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  const [expanded, setExpanded] = React.useState('');
+
+  const handleChangeExpanded = (panel) => (event, newExpanded) => {
+    setExpanded(newExpanded ? panel : false);
+  };
+
+  function getProductParams(product) {
```

Comment:
Znowu pojawia się coś co jest już w innym komponencię. Pomyśl o utworzeniu z tego nowych komponentów bo generalnie to jest jak osobny komponent. to samo z `getParamOptions`

Reply:
Done.